code/ship/shiphit.cpp

```diff
@@ -113,7 +113,7 @@ void do_subobj_destroyed_stuff( ship *ship_p, ship_subsys *subsys, vec3d* hitpos
 	object *ship_objp;
 	model_subsystem *psub;
 	vec3d	g_subobj_pos;
-	int type, i, log_index;
+	int type, i, subsystem_index, log_index;
```

Comment:
If you are renaming the variable you could also move the declaration closer to its usages.

Reply:
*shrug*